### PR TITLE
Adding the preview property from the Reddit API post response to the …

### PR DESCRIPTION
**Pull request by CSedar21**
…post class

### CODE REVIEW

File: RedditSharp/Things/Image.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        /// Url for the image.
+        /// </summary>
+        [JsonProperty("url")]
+        public Uri Url { get; private set; }
+
+        /// <summary>
+        /// Width of the image.
+        /// </summary>
+        [JsonProperty("width")]
+        public int Width { get; private set; }
```
Comment (zmarty) [inline RedditSharp/Things/Image.cs]: Is this always available? If no, set it to nullable int?

File: RedditSharp/Things/Image.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+        /// <summary>
+        /// Height of the image.
+        /// </summary>
+        [JsonProperty("height")]
+        public int Height { get; private set; }
```
Comment (zmarty) [inline RedditSharp/Things/Image.cs]: Is this always available? If no, set it to nullable int?

File: RedditSharp/Things/Image.cs (hunk 1 above)
Comment (CSedar21) [reply RedditSharp/Things/Image.cs]: I updated the property to be int? instead of int

File: RedditSharp/Things/Image.cs (hunk 2 above)
Comment (CSedar21) [reply RedditSharp/Things/Image.cs]: I updated the property to be int? instead of int

**Review by zmarty — APPROVED** [review]

**Pull request merged**